content/en/platform/corda/5.0-dev-preview-2/getting-started/debugging-against-local-corda/debugging-against-local-corda.md

```diff
@@ -10,20 +10,20 @@ section_menu: corda-5-dev-preview2
 ---
 The CSDE includes an IntelliJ run-time configuration which allows you to attach IntelliJ’s debugger to the local Corda cluster.
 {{< note >}}
-Currently there is a time out issue when debugging Corda. As the local Corda cluster is a real Corda cluster, it behaves like one. Specifically, if the workers stop working, Corda detects it and errors out the flow. Unfortunately, it currently mistakes code held at a debugging point as a processor that has stopped working. As a result, although you can halt at a breakpoint and examine the stack, if you wait too long before resuming the execution of the code, it times out and the CorDapp does not continue to run as expected.     
+Currently there is a time-out issue when debugging Corda. As the local Corda cluster is a real Corda cluster, it behaves like one. Specifically, if the workers stop working, Corda detects it and errors out the flow. Unfortunately, it currently mistakes code held at a debugging point as a processor that has stopped working. As a result, although you can halt at a breakpoint and examine the stack, if you wait too long before resuming the execution of the code, it times out and the CorDapp does not continue to run as expected.     
 {{< /note >}}
 To debug:
 1. [Start the Corda combined worker using the startCorda Gradle helper](../running-your-first-cordapp/run-first-cordapp.html#starting-the-corda-combined-worker), if not already started.
 2. [Deploy your CorDapp using deployCordapp](../running-your-first-cordapp/run-first-cordapp.html#deploying-a-cordapp), if not already deployed.
 3. Select the `DebugCorDapp` run configuration.
 {{< figure src="select-debug-configuration.png" figcaption="CSDE DebugCorDapp run configuration in IntelliJ" >}}
-4. Click the debug button.
+4. Click the **Debug** button.
```

Comment:
Should this be bold even though it's not actually written on the interface?

Reply:
Sorry happy for it to be bold if that makes sense to you. More a point for discussion at some point

Reply:
I generally think the title of a part of the interface should be bolded as per the style guide even to make it stand out but yeah can add it to things to discuss.